### DISCUSSION TITLE
passing Data to Widget

### DISCUSSION THREAD

**Discussion by swarakaka** [filamentphp/filament#1580] [Help] [answered]
Hello,
How can i passing Data from View[source] to a Widget ?

**Comment by swarakaka** [comment, marked as answer]
https://filamentphp.com/docs/2.x/admin/resources#building-widgets

```
use Illuminate\Database\Eloquent\Model;
 
public ?Model $record = null;
```

**Reply by massi-dz** [reply]
how use that

**Reply by cubancodepath** [reply]
I try to use that in a custom page, where i get the record by url parameter but not working
